src/amber/cli/generators/scaffold.cr

```diff
@@ -7,17 +7,84 @@ module Amber::CLI
     property view : Generator
 
     def initialize(name, fields)
-      super(name, fields)
-      if config.model == "crecto"
-        @migration = CrectoMigration.new(name, fields)
-        @model = CrectoModel.new(name, fields)
-        @controller = ScaffoldCrectoController.new(name, fields)
+      # Keywords from https://github.com/crystal-lang/crystal/wiki/Crystal-for-Rubyists#available-keywords as of May 6th, 2019
+      keywords = [
```

Comment:
this could prob be a constant `KEYWORDS = [...]`

Reply:
It can't.
```

Syntax error in src/amber/cli/generators/scaffold.cr:11: dynamic constant assignment. Constants can only be declared at the top level or inside other types.

      KEYWORDS = [
```

Reply:
Would have to move it outside of `initialize`.